Dataset_Sheet

```diff
@@ -0,0 +1 @@
+
```

Comment:
![image](https://github.com/user-attachments/assets/884e4795-05ce-4663-98a9-705ebe3e9a6f)
I used the prompt "Vines on a ruined building with vibrant bright sky", I wanted to keep the vibrant colors in the images. This image is an old run-down building showing the corners where the vines are growing into the sky with a bright night sky.